### PR TITLE
(chore)Update Contributing section

### PR DESCRIPTION
**Pull request by elfrank**
## Brief Description
Update "Contributing" section on the README and wiki page.

Specify we are looking to improve test coverage and Code Climate score.

## Pull Request Guidelines

- [x] I have added pull requests labels which describe my contribution.
- [x] All existing tests passed.
    - [x] I have added tests to cover my changes, of which pass.
- [x] I have [compared](https://github.com/hoverinc/ray-tracing-renderer/wiki/Contributing#comparing-changes) the render output of my branch to `master`.
- [x] My code has passed the ESLint configuration for this project.
- [x] My change requires modifications to the documentation.
    - [x] I have updated the documentation accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,6 +64,15 @@ Ray Tracing Renderer runs on WebGL2, and does so by implementing a [path tracing
...
+## Contributing
+We want to increase test coverage and maintanability of the repo. If you would like to contribute, take a look at the following and submit Pull Requests:
+* [CodeClimate issues](https://codeclimate.com/github/hoverinc/ray-tracing-renderer/issues) to improve maintainability, and
+* Unit tests to [improve our coverage](https://codeclimate.com/github/hoverinc/ray-tracing-renderer/code)
+
+Take a look to [this page](https://github.com/hoverinc/ray-tracing-renderer/wiki/Submitting-Changes) with more details about submitting changes to the project.
+
+### Expectations
+
+This repository started as a side-project and the time we invest on it is limited. It make take us a few days to get back to you but please bring your ideas forward. We'll do our best to respond promptly.
```
Comment (lyonsno) [inline README.md:78]: `nit` It make take us a few days => it may take us a few days

Good call adding that disclaimer

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 # Ray Tracing Renderer
 A [Three.js](https://github.com/mrdoob/three.js/) renderer which utilizes path tracing to render a scene with true photorealism. The renderer supports global illumination, reflections, soft shadows, and realistic environment lighting.
 
-[Demo](https://hoverinc.github.io/ray-tracing-renderer/scenes/sample-models/) | [User Guide](https://github.com/hoverinc/ray-tracing-renderer/wiki/User-Guide) | [API Reference](https://github.com/hoverinc/ray-tracing-renderer/wiki/RayTracingRenderer) | [Contributing](https://github.com/hoverinc/ray-tracing-renderer/wiki/Contributing)
+[Demo](https://hoverinc.github.io/ray-tracing-renderer/scenes/sample-models/) | [User Guide](https://github.com/hoverinc/ray-tracing-renderer/wiki/User-Guide) | [API Reference](https://github.com/hoverinc/ray-tracing-renderer/wiki/RayTracingRenderer) | [Contributing](https://github.com/hoverinc/ray-tracing-renderer#contributing)
```
Comment (lyonsno) [inline README.md:7]: The contributing link seems to be broken? When I click on it it just brings me to the main repo page.

Clicking the `contributing` link on the repo page takes me to a page asking me if I want to create a `contributing` section.

**Review by lyonsno — CHANGES_REQUESTED** [review]
This is a good contribution to the repo for supporting open-source, everything looks great except the issue with the contributing page.

File: README.md (hunk 1 above)
Comment (elfrank) [reply README.md]: good catch, fixed the typo

File: README.md (hunk 2 above)
Comment (elfrank) [reply README.md:7]: The link https://github.com/hoverinc/ray-tracing-renderer#contributing doesn't exist in master yet. Once the PR gets merged, the link should work. The same way the following page works: https://github.com/hoverinc/ray-tracing-renderer/#introduction

**Review by lyonsno — APPROVED** [review]
Ok, LGTM then!

**Pull request merged**